.gitignore

```diff
@@ -9,8 +9,7 @@ bindings/java/foundationdb-tests*.jar
 bindings/java/fdb-java-*-sources.jar
 packaging/msi/FDBInstaller.msi
 build/
-cmake-build-debug/
-cmake-build-release/
+cmake-build-*/
```

Comment:
Do we intend to check this change?

Reply:
I think it's really useful for Clion users because Clion always generates cmake-build-*/ directories based on the compiler configuration name. If don't use `.gitignore`, we always need to be careful when adding files to commit and cannot use `git add --all` for convenience.